### PR TITLE
[US_ND] Admissions from Supervision to Prison by officer chart

### LINKED ISSUE #1612: [US_ND][Fast follow] Admissions from Supervision to Prison by officer chart
**Is your feature request related to a problem? Please describe.**
We will add a new chart to the "Supervision to Prison" page. This chart uses a new metric file, `supervision_to_prison_population_snapshot_by_officer` which is in the `jovergaag/1556-fixtures` branch if it hasn't already been merged to main. The fixture looks very similar to the `supervision_to_prison_population_snapshot_by_dimension`, but there will be lots of officer names so we had to reduce the number of dimensions (filters) supported so that the file doesn't get too big.

The enabled filters will be: TIME_PERIOD, AGE_GROUP, DISTRICT, SUPEVISION_TYPE, GENDER, SUPERVISION_LEVEL, RACE

accessor:`officer_name`
enableMetricModeToggle: true

Enable the chart only for US_ND

[Design](https://www.figma.com/file/Mw6kSEP2jdr9i7UVEjPuCI/UP-Consolidation?node-id=3519%3A6675)

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
Add any other context about the feature request here.

### PR DESCRIPTION
**Pull request by nasaownsky**
## Description of the change

Added Admissions from Supervision to Prison by officer chart

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Closes #1612 

### Development

These boxes should be checked by the submitter prior to merging:

- [ ] Manual testing against realistic data has been performed locally
- [ ] E2E have been run for Lantern changes ([Steps in the Readme](https://github.com/Recidiviz/pulse-dashboard#running-e2e-tests))

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Comment by nasaownsky** [comment]
The chart isn't display. May it be problem with the data? I checked metadata and `time_period` manifest is different in `supervision_to_prison_population_snapshot_by_officer` than in `supervision_to_prison_population_snapshot_by_dimension`, but replacing it doesn't change anything.

File: src/tenants.ts [hunk 1]
```diff
@@ -363,6 +363,7 @@ const TENANTS: Tenants = {
         // PATHWAYS_SECTIONS.countBySupervisionLevel,
         PATHWAYS_SECTIONS.countByGender,
         PATHWAYS_SECTIONS.countByRace,
+        PATHWAYS_SECTIONS.countByOfficer,
```
Comment (jovergaag) [inline src/tenants.ts:366]: Please move this up to be just below `countByLocation`

**Review by jovergaag — COMMENTED** [review]
I started to debug it but ran out of time. I'll take a look again in the mormning.

**Comment by jovergaag** [comment]
It might be best to create a new metric for this since there are so many differences in dimensions between it and `SupervisionPopulationSnapshotMetric`

**Comment by nasaownsky** [comment]
Yeah I tried, but no data appeared. In #1674 snapshot charts also wasn't work, but changed `time_period` in fixture made it work there. Here it isn't affect chart. Maybe another problem in fixtures?

**Comment by jovergaag** [comment]
@nasaownsky You were right, the fixtures were missing an important value. It is fixed now and I'm seeing data in the chart. There is still some formatting to do to make it look pretty. I'll review it once that formatting is all finished!

**Comment by jovergaag** [comment]
Also, please sort them from the highest count to the lowest, with the highest on the left of the X-axis, and only show the 20 officers with the highest counts.

File: src/core/VizPopulationSnapshot/VizPopulationSnapshot.tsx [hunk 2]
```diff
@@ -115,6 +117,8 @@ const VizPopulationSnapshot: React.FC<VizPopulationOverTimeProps> = ({
 
   sortByLabel(data, "accessorLabel");
 
+  if (isOfficer) sortByValues(data, "value").splice(20);
```
Comment (jovergaag) [inline src/core/VizPopulationSnapshot/VizPopulationSnapshot.tsx:120]: I've shown this to the design team, and they have decided to change the requirement. We will not show the officer name on the x-axis, and only show the name in the tooltip. Do not splice it to the highest 20 officers.

This allows us to show all officers - specifically the highest revocation count officers, AND the lowest revocation count officers which is also important. Sorry for the changes.

**Review by jovergaag — CHANGES_REQUESTED** [review]
Looks good other than those quick changes based on requirements change.

**Review by jovergaag — COMMENTED** [review]
This looks great now! As with the other ND chart, I'm waiting for sign-off from data quality and product. Once that is in I will approve and merge!

File: src/core/models/utils.ts [hunk 3]
```diff
@@ -181,6 +182,7 @@ export function createSupervisionPopulationSnapshot(
           ],
         supervisionLevel: record.supervision_level,
         race: record.race,
+        officerName: record.officer_name,
```
Comment (jovergaag) [inline src/core/models/utils.ts:185]: Please format this name so that it is title case - feedback from design review.

**Review by jovergaag — CHANGES_REQUESTED** [review]
Design review uncovered a couple of things. The download data is broken because of a missing filter option, and we need to change the formatting of the name to be title case.

**Review by jovergaag — APPROVED** [review]
Done, finally :)

**Pull request merged**